catlin/pkg/cmd/validate/validate.go

```diff
@@ -84,10 +87,18 @@ func validate(cli app.CLI, path string) error {
 	// print result
 	out := cli.Stream().Out
 	for _, v := range result.Lints {
-		if v.Kind == validator.Error {
+		switch v.Kind {
+		case validator.Error:
 			fmt.Fprintf(out, "ERROR: %s\n", v.Message)
-		} else {
+		case validator.Warning:
+			fmt.Fprintf(out, "WARN : %s\n", v.Message)
+		case validator.Recommendation:
```

Comment:
unrelated to this patch: perhaps we should rename `Recommendation` to `Hint` :D

Reply:
yeah :smiley_cat: will address in a separate patch